src/pykeen/losses.py

```diff
@@ -259,11 +259,13 @@ def process_lcwa_scores(
     ) -> torch.FloatTensor:
         """Process scores from LCWA training loop."""
         # Apply label smoothing
-        if label_smoothing is not None and num_entities is not None and label_smoothing > 0.:
+        if label_smoothing is not None and label_smoothing > 0.:
```

Comment:
can we move this part into the `apply_label_smoothing` method? iirc we had similar code in other places, too.

Reply:
Yeah that would be fine

Reply:
5636fa9b